main.md

```diff
@@ -178,7 +178,7 @@ The JSON objects with `type` fields of `account_information` and `payment_initia
 The allowable contents of the authorization details object are determined by the `type` parameter.
 
 `type`:
-:   The type of resource request as a string. This field MAY define which other elements are allowed in the request. This element is REQUIRED.
```

Comment:
@jricher I assume this was a CnP error?

Reply:
This was actually intentional, as it's type type for the whole request object. That said, I'm fine with the change.